### PR TITLE
Feature/create new target service

### PR DESCRIPTION
**Pull request by mwtz**
## What & Why:

I should be able to create a new target so I can find people with similar interests in a specific area

image

image

## Risks, Mitigation & Rollback:

- [ ] Safe to Revert *check this box if this PR can be reverted without incident*

### CODE REVIEW

File: src/components/mapView/index.js [hunk 1]
```diff
@@ -1,26 +1,40 @@
...
+      click: e => {
+        const { lat, lng } = e.latlng;
+        history.push({
+          pathname: routesPaths.newtarget,
+          search: `?lat=${lat}&lng=${lng}`,
+        });
+      },
+    });
+    return null;
+  };
```
Comment (diebarral) [inline src/components/mapView/index.js:37]: I think you just need: 

```
const map = useMapEvents({
      click: e => {
        const { lat, lng } = e.latlng;
        history.push({
          pathname: routesPaths.newtarget,
          search: `?lat=${lat}&lng=${lng}`,
        });
      },
    });
```

According to [React Leaflet](https://react-leaflet.js.org/docs/example-events/)

File: src/utils/topics.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+ICONS_MAP.set(11, emptyOval);
+ICONS_MAP.set(12, emptyOval);
+ICONS_MAP.set(2, footballIcon);
+ICONS_MAP.set(13, politicsIcon);
+ICONS_MAP.set(14, artIcon);
+ICONS_MAP.set(15, datingIcon);
+ICONS_MAP.set(16, musicIcon);
+ICONS_MAP.set(17, moviesIcon);
+ICONS_MAP.set(19, foodIcon);
+ICONS_MAP.set(18, seriesIcon);
```
Comment (diebarral) [inline src/utils/topics.js]: You can pass an iterable to `new Map([[1, 'a'], [2, 'b']]);`. You can[ find more here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/Map)

**Review by diebarral — APPROVED** [review]

File: src/components/form/select/index.js [hunk 3]
```diff
@@ -10,7 +10,7 @@ const Select = ({ register, options, name, error, placeholder, className = '', .
         {placeholder}
       </option>
       {options.map((value, i) => (
-        <option key={i} value={value}>
+        <option key={i} value={JSON.stringify(value)}>
```
Comment (asdolo) [inline src/components/form/select/index.js:13]: Why is this change needed?

**Review by asdolo — APPROVED** [review]
